tests/cross_functional/ui/test_validation_ui.py

```diff
@@ -108,12 +108,7 @@ def test_odf_cephblockpool_compression_status(self, setup_ui_class_factory):
         """
         setup_ui_class_factory()
 
-        storage_system_details = (
-            ValidationUI()
-            .nav_storage_cluster_default_page()
-            .nav_storage_systems_tab()
-            .nav_storagecluster_storagesystem_details()
-        )
+        storage_system_details = ValidationUI().nav_storage_cluster_default_page()
```

Comment:
storage_system_details does not exist in 4.20. if we change it lets rename to `block_and_file_page`.

Reply:
Fixed